cms-contentful/it/ContentfulCmsServiceIT.java

```diff
@@ -0,0 +1,68 @@
+import io.commercetools.sunrise.cms.CmsIdentifier;
+import io.commercetools.sunrise.cms.contentful.ContentfulCmsService;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.Optional;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.TimeUnit;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ContentfulCmsServiceIT {
+    private static final Locale DE = Locale.forLanguageTag("de");
+    private static final Locale EN_US = Locale.forLanguageTag("en-US");
+    private static final List<Locale> SUPPORTED_LOCALES = asList(DE, EN_US);
+
+    // credentials for contentful demo account
+    private static final String SPACE_ID = "cfexampleapi";
```

Comment:
So this and another comment below are about the same topic: the test data. So @dariuszor is using the public demo project that Contentful is using for their own API call examples. It is a good idea as a short-term, but for the long run it totally depends on the stability they can assure us for that project and I'm not particularly aware of it. The public project has the same data that they provide for any project if you request it.
I'd say, let's work with the demo project for now, but pass the project credentials as environment vars, as if it were a generic project. Once we decide to move for the option of creating and setting up your own project, we only need to create the corresponding test fixtures and nothing else.